plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseConnectorSmokeTest.java

```diff
@@ -53,4 +55,46 @@ public void testShowCreateTable()
                         "   engine = 'LOG'\n" +
                         ")");
     }
+
+    @Override
+    @Test
+    public void testDeleteAllDataFromTable()
+    {
+        // Override because default storage engine doesn't support delete
+        String tableDefinition = "(" +
+                " regionkey BIGINT NOT NULL," +
+                " name VARCHAR(25) NOT NULL," +
+                " comment VARCHAR(115) NOT NULL" +
+                "    ) WITH (engine='MergeTree',  order_by=ARRAY['regionkey'])";
+
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_delete_all_data", tableDefinition)) {
+            assertQuerySucceeds(format("insert into %s select * from region", table.getName()));
```

Comment:
not possible create table with "as select * from region".  default  clickhouse engine in test(log) does not support delete